help/view-and-edit-your-message-drafts.md

```diff
@@ -69,6 +71,27 @@ saved for 30 days.
 
 {end_tabs}
 
+### Delete multiple drafts
+
+{start_tabs}
+
+{tab|desktop-web}
+
+{!go-to-draft-messages.md!}
+
+1. Click **Select all drafts** near the upper right corner of the drafts view,
+   or select the drafts you want to delete by toggling the checkboxes.
+
+1. Click the **trash** (<i class="fa fa-trash-o"></i>) icon near the upper right
+   corner of the drafts view to delete all selected drafts.
+
+!!! keyboard_tip ""
+
+    You can also use <kbd>Backspace</kbd> within the drafts view to delete the
+    selected draft.
+
+{end_tabs}
```

Comment:
I think this tip should be dropped here, since it doesn't apply to multiple drafts.